src/figment.rs

```diff
@@ -21,10 +21,19 @@ use crate::coalesce::{Coalescible, Order};
 ///
 /// ## Merging vs. Joining
 ///
-/// _Merging_, _joining_, and _adjoining_ control whether duplicate values are
-/// replaced, discarded, or concatened. A _merged_ value replaces an existing
-/// value with the same key, while a _joined_ value is discarded if a value with
-/// the same key exists:
+/// _Merging_, _joining_, _admerging_ and _adjoining_ control whether duplicate
+/// values are replaced, discarded, or concatenated. A _merged_ value replaces
+/// an existing value with the same key, while a _joined_ value is discarded if
+/// a value with the same key exists.
+///
+/// | Method                 | Data type | Result              |
```

Comment:
I like the idea behind this table, but I'm not sure if this is quite the representation that would be most helpful. I would expect exactly four rows, one for each strategy, and for the columns to tell me what each strategy does in a given case. Perhaps the columns should be: strategy (merge, join, adjoin, admerge) linking to each relevant method, array behavior, dict behavior, non-composite value behavior. I'm not sure if "behavior" is the correct word, but we effectively want to say that we keep a previous value, new value, or concatenate, so whatever collectively captures those ideas is the word we want to use for those columns.

Reply:
You're right! I've updated the table exactly like you describe in 28c86af.